File: fix.py
import re
with open('index.html', 'r', encoding='utf-8') as f:
    c = f.read()

# Replace the top header buttons
c = re.sub(
    r'<div style="display:flex;gap:5px">\s*<button class="btn bp bsm" onclick="openSupExportFromCard\(\)">(.*?)<\/button>\s*<button class="btn bo bsm" onclick="sucToggleEdit\(\)">(.*?)<\/button>\s*<button class="btn bs bsm" onclick="CM\(''sucard-m''\)">(.*?)<\/button>\s*<\/div>',
    r'<div style="display:flex;gap:5px">\n          <button id="suc-btn-exp-purch" class="btn bg bsm" onclick="sucExportDocs()" style="display:none">?? יצוא דוח רכש</button>\n          <button id="suc-btn-exp-act" class="btn bp bsm" onclick="openSupExportFromCard()" style="display:none">?? יצוא דוח פעילות</button>\n          <button class="btn bo bsm" onclick="sucToggleEdit()">\2</button>\n          <button class="btn bs bsm" onclick="CM(''sucard-m'')">\3</button>\n        </div>',
    c, flags=re.DOTALL)

with open('index.html', 'w', encoding='utf-8') as f:
    f.write(c)

File: fix.js
const fs = require('fs');
let c = fs.readFileSync('index.html', 'utf8');
const search = '<div class="vbtns" style="overflow-x:auto">\\r\\n              <button id="vlb-group-pairs-mobile" class="vbtn active" onclick="setListGroupMode(\\'pairs\\')">⚖️ זוגות</button>\\r\\n              <button id="vlb-group-clusters-mobile" class="vbtn" onclick="setListGroupMode(\\'clusters\\')">📦 אשכולות</button>\\r\\n            </div>';
const replace = '<div class="vbtns" style="overflow-x:auto; display:flex; align-items:center; flex:1;">\\r\\n              <button id="vlb-group-pairs-mobile" class="vbtn active" onclick="setListGroupMode(\\'pairs\\')">⚖️ זוגות</button>\\r\\n              <button id="vlb-group-clusters-mobile" class="vbtn" onclick="setListGroupMode(\\'clusters\\')">📦 אשכולות</button>\\r\\n              <button class="btn bp bsm" onclick="openNewSched()" style="margin-right:auto; white-space:nowrap; padding:4px 10px; font-size:0.8rem; border-radius:15px; box-shadow:0 1px 3px rgba(0,0,0,0.2);">➕ שיבוץ חדש</button>\\r\\n            </div>';
c = c.replace(search, replace);
fs.writeFileSync('index.html', c);
